packet.go

```diff
@@ -63,6 +69,10 @@ type Packet struct {
 
 	// TargetIP specifies the IPv4 address of the target of this Packet.
 	TargetIP net.IP
+
+	// RemoteAddr specifies the actual remote address from which the
+	// ARP packet was received.
```

Comment:
Can we add another note here that says something along the lines of: "RemoteAddr does not appear in the binary form of a Packet".